app/javascript/styles/components.scss

```diff
@@ -3343,3 +3343,7 @@ button.icon-button.active i.fa-retweet {
   transform: translate(-50%, -50%);
 }
 /* End Video Player */
+
+.notification-bar {
```

Comment:
`.notification-bar` is the class name of [`react-notification`](https://github.com/pburtchaell/react-notification). It may be overwritten.

I think it would be better to add `activeBarStyle` to [`./app/javascript/mastodon/selectors/index.js` Line 59](https://github.com/tootsuite/mastodon/blob/ef226a6f223ad3bdf662efb0ed75694b3f0bfb56/app/javascript/mastodon/selectors/index.js#L59)